Geometry/CMSCommonData/data/caloBase/2026/v7/caloBase.xml

```diff
@@ -0,0 +1,337 @@
+   </MaterialFraction>
+  </CompositeMaterial>
+</MaterialSection>
+
+<SolidSection label="caloBase.xml">
+  <Polycone name="CALO" startPhi="0*deg" deltaPhi="360*deg">
+    <ZSection z="-[cms:CalorBeamZ2]" rMin="[cms:CalorMuonR2]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="-[cms:CalorBeamZ3]" rMin="[cms:CalorMuonR2]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="-[cms:CalorBeamZ3]" rMin="[cms:CalorBeamR2]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="-[cms:CalorBeamZ1]" rMin="[cms:CalorBeamR1]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="-[cms:TrackBeamZ2]" rMin="[cms:TrackBeamR2]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="-[cms:TrackBeamZ2]" rMin="[cms:TrackCalorR]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="[cms:TrackBeamZ2]"  rMin="[cms:TrackCalorR]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="[cms:TrackBeamZ2]"  rMin="[cms:TrackBeamR2]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="[cms:CalorBeamZ1]"  rMin="[cms:CalorBeamR1]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="[cms:CalorBeamZ3]"  rMin="[cms:CalorBeamR2]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="[cms:CalorBeamZ3]"  rMin="[cms:CalorMuonR2]" rMax="[cms:CalorMuonR]"/>
+    <ZSection z="[cms:CalorBeamZ2]"  rMin="[cms:CalorMuonR2]" rMax="[cms:CalorMuonR]"/>
+  </Polycone>
+  <Polycone name="CALOEC" startPhi="0*deg" deltaPhi="360*deg">
+    <ZSection z="[Zpos00]" rMin="[Rmin00]" rMax="[Rmax00]"/>
+    <ZSection z="[Zpos01]" rMin="[Rmin00]" rMax="[Rmax01]"/>
+    <ZSection z="[Zpos01]" rMin="[Rmin00]" rMax="[Rmax01H]"/>
+    <ZSection z="[Zpos02]" rMin="[Rmin00]" rMax="[Rmax02]"/>
+    <ZSection z="[Zpos03]" rMin="[Rmin00]" rMax="[Rmax03]"/>
+    <ZSection z="[Zpos03]" rMin="[Rmin04]" rMax="[Rmax03]"/>
+    <ZSection z="[Zpos04]" rMin="[Rmin04]" rMax="[Rmax04]"/>
+    <ZSection z="[Zpos05]" rMin="[Rmin04]" rMax="[Rmax04]"/>
+  </Polycone>
+  <Box name="CaloECNotch" dx="[XwidNotch]" dy="[YwidNotch]" dz="[ZwidNotch]"/>
+  <Polycone name="CALOECTSFront" startPhi="0*deg" deltaPhi="360*deg">
+    <ZSection z="[Zpos0L]" rMin="[Rmin10]" rMax="[Rpos0L]"/>
+    <ZSection z="[Zpos0H]" rMin="[Rmin10]" rMax="[Rpos0H]"/>
+    <ZSection z="[Zpos0H]" rMin="[Rmin10]" rMax="[Rpos1L]"/>
+    <ZSection z="[ZposV0]" rMin="[Rmin10]" rMax="[RposV0]"/>
+  </Polycone>
+  <Polycone name="CALOECFront" startPhi="0*deg" deltaPhi="360*deg">
+    <ZSection z="[Zpos10]" rMin="[Rmin11]" rMax="[Rmax10]"/>
+    <ZSection z="[Zpos0H]" rMin="[Rmin11]" rMax="[Rmax11]"/>
+    <ZSection z="[ZposV0]" rMin="[Rmin11]" rMax="[Rmax11]"/>
+  </Polycone>
+  <Polycone name="CALOECTSModerator0" startPhi="0*deg" deltaPhi="360*deg">
+    <ZSection z="[ZposAL]" rMin="[Rmin11]" rMax="[RposAL]"/>
+    <ZSection z="[ZposAH]" rMin="[Rmin11]" rMax="[RposAH]"/>
+    <ZSection z="[ZposV0]" rMin="[Rmin11]" rMax="[RposAH]"/>
+  </Polycone>
+  <SubtractionSolid name="CALOECTSModerator1">
+    <rSolid name="CALOECTSModerator0"/>
+    <rSolid name="CaloECNotch"/>
+    <Translation x="[XposNotch]" y="0.*fm" z="[ZposNotch]"/>
```

Comment:
In future, could we please use mm for length units and not fm? It's harmless here since it is 0, but in general I don't think there is a benefit to using fm instead of mm in the geometry.